server/src/main/java/io/druid/server/lookup/cache/LookupCoordinatorManagerConfig.java

```diff
@@ -26,63 +26,45 @@
 
 public class LookupCoordinatorManagerConfig
 {
-  public static final Duration DEFAULT_HOST_DELETE_TIMEOUT = Duration.millis(1_000L);
-  public static final Duration DEFAULT_HOST_UPDATE_TIMEOUT = Duration.millis(10_000L);
-  public static final Duration DEFAULT_DELETE_ALL_TIMEOUT = Duration.millis(10_000L);
-  public static final Duration DEFAULT_UPDATE_ALL_TIMEOUT = Duration.millis(60_000L);
-  @JsonProperty
-  private Duration hostDeleteTimeout = null;
-  @JsonProperty
-  private Duration hostUpdateTimeout = null;
+  public static final Duration DEFAULT_HOST_TIMEOUT = Duration.millis(2_000L);
+  public static final Duration DEFAULT_ALL_HOST_TIMEOUT = Duration.millis(900_000L);
+
   @JsonProperty
-  private Duration deleteAllTimeout = null;
+  private Duration hostTimeout = null;
+
   @JsonProperty
-  private Duration updateAllTimeout = null;
+  private Duration allHostTimeout = null;
+
   @JsonProperty
-  @Min(1)
+  @Min(2) //minimum 2 threads, one for lookupManagementLoop and one for talking to the lookup nodes
   private int threadPoolSize = 10;
+
   @JsonProperty
   @Min(1)
-  private long period = 30_000L;
-
-  public Duration getHostDeleteTimeout()
-  {
-    return hostDeleteTimeout == null ? DEFAULT_HOST_DELETE_TIMEOUT : hostDeleteTimeout;
-  }
-
-  public void setHostDeleteTimeout(Duration hostDeleteTimeout)
-  {
-    this.hostDeleteTimeout = hostDeleteTimeout;
-  }
+  private long period = 120_000L;
```

Comment:
nothing specific but every 2 mins sync up should be good enough , users can adjust if necessary.